pkg/utils/kube/ctl.go

```diff
@@ -148,17 +134,14 @@ func (k *KubectlCmd) LoadOpenAPISchema(config *rest.Config) (openapi.Resources,
 	if err != nil {
 		return nil, nil, fmt.Errorf("error getting openapi resources: %s", err)
 	}
-	gvkParser, err := newGVKParser(oapiGetter)
+	gvkParser, err := k.newGVKParser(oapiGetter)
 	if err != nil {
-		// return a specific error type to allow gracefully handle
-		// creating GVK Parser bug:
-		// https://github.com/kubernetes/kubernetes/issues/103597
-		return oapiResources, nil, NewCreateGVKParserError(err)
+		return oapiResources, nil, err
```

Comment:
Please add context in the returned error so we know which code path it came from.